metadata/cube.xml

```diff
@@ -11,17 +11,17 @@
 			<option name="activate" type="button">
 				<_short>Activate</_short>
 				<_long>Activates the cube with the specified button.</_long>
-				<default>&lt;alt&gt; &lt;ctrl&gt; BTN_LEFT</default>
+				<default>&lt;super&gt; &lt;shift&gt; BTN_LEFT</default>
 			</option>
 			<option name="rotate_left" type="activator">
 				<_short>Rotate left</_short>
 				<_long>Rotates left with the specified activator.</_long>
-				<default>&lt;alt&gt; &lt;ctrl&gt; KEY_LEFT</default>
+				<default>&lt;super&gt; &lt;ctrl&gt; KEY_H</default>
```

Comment:
You should either keep the left/right bindings as they are or just disable them.

Reply:
It matches well with the config as a whole.

https://github.com/WayfireWM/wayfire/wiki/Tutorial#default-key-bindings